docs/digger/suggestions

```diff
@@ -24,4 +24,6 @@ HTTP Proxy Usage Limitations: Can we refer to some other documentation regarding
 - should widely used terms such as:
   .Ansible Playbook
   .Jenkins Plugin
-  etc. be CAPITALISED?
\ No newline at end of file
+  etc. be CAPITALISED?
+  
+- don't use "machine", use "system" instead. Machine sounds very "mechanical". ????
```

Comment:
?

Reply:
@PhilipGough  I believe this file is being used by @pmacko1 to help with writing the documentation. I don't want to remove it altogether but I think it should be added to .gitignore by @pmacko1 so he can keep it locally.